src/main/java/org/opensearch/geospatial/geojson/Feature.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.geojson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Feature {
+    public static final String TYPE = "Feature";
+    private final Map<String, Object> geometry;
+    private Map<String, Object> properties = new HashMap<>();
+
+    private Feature(Map<String, Object> geometry) {
+        this.geometry = geometry;
+    }
+
+    public Map<String, Object> getGeometry() {
```

Comment:
Given this is a new repo, I think you have an opportunity to set and enforce code style. For instance, when will you want to ensure there are javadocs?

Reply:
Agree. I created an issue for myself to work on this https://github.com/opensearch-project/geospatial/issues/17